packages/ui/src/button/Button.tsx

```diff
@@ -96,8 +86,8 @@ const Button = ({
     L: 'rounded-[26px] w-[100%] h-[52px] box-border',
   }
   const variantToClass = {
-    primary: `w-[100%] text-white box-border ${backgroundColors[color]}`,
-    outline: `w-[100%] ${textColors[color]} border ${borderColors[color]} bg-transparent`,
+    primary: `w-[100%] text-white box-border ${backgroundColors[getColor(color, warning)]}`,
+    outline: `w-[100%] ${textColors[getColor(color, warning)]} border ${borderColors[getColor(color, warning)]} bg-transparent`,
```

Comment:
```suggestion
    cons warningColor = warning ? 'darkRed' : color
    ...
    outline: `w-full ${textColors[warningColor]} border ${borderColors[warningColor]} bg-transparent`,
```
글고 요거 이런식으로 변수 바로 넣어서 하면 안돼요 줜님꺼에서는??
제가 해봣는데 되는 거 같아여

Reply:
됩니당!! ㅎㅎ 함수로만 생각했는데 더 간단한 방법이 있었네요 히히